build.gradle

```diff
@@ -989,6 +989,11 @@ project(':iceberg-spark3') {
   apply plugin: 'scala'
 
   sourceSets {
+    main {
```

Comment:
@rdblue never-mind this IS required if you want to do cross compilation (Have Java code depend on Scala code in the same module)

Reply:
Do we need the Java plugin if we are using Scala here? I'm wondering why we were using both in the past. Maybe we only had Scala depending on Java code.

Reply:
Yeah the other direction is fine, we don't have the Java plugin set for this project but the Scala plugin does extends the Java one.